lab-ron-dunphy/lib/read-files.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+
+var fileArray = [];
+
+module.exports = function(callback) {
+
+  fs.readFile('one.txt', function(err, data){
+    if(err) {callback(err);}
+    fileArray.push(data.toString('hex', 0, 8));
+
+    fs.readFile('two.txt', function(err, data){
+      if(err) {callback(err);}
+      fileArray.push(data.toString('hex', 0, 8));
+
+      fs.readFile('three.txt', function(err, data){
+        if(err) {callback(err);}
+        fileArray.push(data.toString('hex', 0, 8));
+        callback(fileArray);
```

Comment:
This is all set up well! Your entry point, index.js, is just not passing in a callback, so your final data isn't actually going anywhere.